packages/lodestar-cli/src/cmds/beacon/cmds/run/run.ts

```diff
@@ -34,7 +36,13 @@ export async function runHandler(options: IBeaconOptions): Promise<void> {
 
   const config = getBeaconConfig(options.preset, options.params);
   const libp2p = await createNodeJsLibp2p(peerId, options.network);
-  const logger = new WinstonLogger();
+  const loggerTransports = [
```

Comment:
We should merge master into dapplion/e2e+ so we don't get these diffs here

Reply:
What about merging `dapplion/e2e+` as skipped and un skip them in the PR that is able to fix it's issues?